src/main.cpp

```diff
@@ -2288,6 +2314,26 @@ DisconnectResult DisconnectBlock(const CBlock& block, const CBlockIndex* pindex,
                 const CTxInUndo &undo = txundo.vprevout[j];
                 if (!ApplyTxInUndo(undo, view, out))
                     fClean = false;
+
+                if (fUpdateAddressIndex) {
```

Comment:
I don't want to do this, because then, to be consistent, I'll have to add similar comments in many places. I'm thinking this PR's comments can serve as second-level documentation. Let me know if you disagree strongly.

Reply:
@LarryRuane I think we need a comment of some sort in the code, to at least link to the PR itself otherwise users won't know where to look unless they do a git blame and search commit history.  One of the issues about relying on Github is that conversations on PRs can be hidden from view (see pic) and no guarantee that the repo stays up.
![image](https://user-images.githubusercontent.com/194979/54235324-3a164300-44ce-11e9-94c1-53205593791d.png)

Reply:
I'm actually inclined to not include comments like this, since the code could easily become cluttered with them. We don't usually include the origin of particular pieces of code.

Reply:
The comments can be removed in the future.  For now, I think they're quite useful in helping to clearly identify explorer related code, especially as the changes are being integrated/assembled over multiple PRs.